pkg/cloudprovider/providers/aws/aws_test.go

```diff
@@ -63,3 +71,148 @@ func TestGetVPCIDFromSubnetList(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateClusterInfra(t *testing.T) {
+	mockCF := &mocks.CloudFormationAPI{}
+	c := &Cloud{
+		cf: mockCF,
+	}
+
+	stacks := []*cloudformation.Stack{
+		{
+			StackName: aws.String("keto-foo-infra"),
```

Comment:
It's worth DRYing up `aws.String("keto-foo-infra")` into a variable as it's being used quite a bit here.